code/modules/events/undead_event.dm

```diff
@@ -8,7 +8,7 @@
 	RS.lightsoutAmount = pick(2,2,3)
 	RS.start()
 	RS.kill()
-	for(var/area/A)
+	for(var/area/A in world)
```

Comment:
```suggestion
	for(var/area/A in GLOB.all_areas)
```

Reply:
I knew we had to have a glob for this, thanks